tests/unit/networked_attachLocalTemplate.test.js

```diff
@@ -0,0 +1,80 @@
+/* global assert, process, setup, suite, test, teardown, sinon */
+require('aframe');
+var helpers = require('./helpers');
+var naf = require('../../src/NafIndex');
+
+require('../../src/components/networked');
+
+suite('networked attachLocalTemplate:false', function() {
+  var scene;
+  var entity;
+  var networked;
+
+  function initScene(done) {
+    var opts = {
+      assets: [
+        "<template id='t1'><a-entity><a-entity class='template-child'></a-entity></a-entity></template>"
+      ],
+      entity: '<a-entity id="test-entity" networked="template:#t1;attachLocalTemplate:false;" position="1 2 3" rotation="4 3 2"><a-box></a-box></a-entity>'
+    };
+    scene = helpers.sceneFactory(opts);
+    naf.utils.whenEntityLoaded(scene, done);
+  }
+
+  function MockNetworkAdapter() {
```

Comment:
This exists in `networked.test.js` and `NetworkConnection.test.js` as well. I would move it to `helpers.js` and reuse.